### ISSUE TITLE
[Suggestion] Boolean combinator operators

### ISSUE THREAD

**Issue by Vidarls** [freya-fs/freya#188]
I find myself combining different `Freya<bool>` computation quite frequently and I have made the following custom operators for myself:

```f#
let (<&>) = Freya.map2 (&&)
let (<||>) = Freya.map2 (||)
```

Might be useful to include as part of the standard operators?

**Comment by kolektiv** [comment]
Yes, seemingly reasonable to me. I don't see why not! I'd probably make `and` = `<&&>` for consistency, but other than that, sure :smile: Would be happy to take a pull request if you get there before me!
